src/parser/core/EventsNormalizer.ts

```diff
@@ -14,6 +14,13 @@ abstract class EventsNormalizer extends Module {
    */
   abstract normalize(events: AnyEvent[]): AnyEvent[];
 
+  /**
+   * Override this value to true to designate an event normalizer to not be run during the
+   * CombatLogParser's normalize function. These normalizers will instead to used on the final
+   * event history (including fabricated events) which is provided to the APL checker.
+   */
+  isAplNormalizer: boolean = false;
+
```

Comment:
i am not a fan of special casing this. let me look at what the underlying issue is and see what i can figure out.

Reply:
Ok, I'll get rid of this. I can fix it by just not applying Channeling to Fists of Fury (the BeginChannel/EndChannel events it adds aren't used anyways).